### DISCUSSION TITLE
Limit user space / user roles

### DISCUSSION THREAD

**Discussion by Havouza15** [immich-app/immich#6291] [General]
Hi!
I have just started with Immich and so far I love it. 
I have a couple of friends that also want to use it but as I see it 2 things is missing for that
1. If I create a new user for them they will be admins with access to everything. That is not good
2. I cant limit the space per user.
3. I dont want to show the total available space for the non admin users.
Is there anything of this in the pipeline, or it is something I have missed

**Comment by bo0tzz** [comment]
1. What do you mean by 'access to everything'? User accounts are separated and each has their own library
2. #1642
3. There is no option for this at the moment.

**Reply by zackpollard** [reply]
I believe 2 and 3 will both be resolved by the same quotas PR #4471

**Reply by Havouza15** [reply]
I see the difference no on nr 1. Misssed before
Then I wait for an update for 2 and 3. Thanks a lot
